docs/how-to/use-existing-private-key.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Use an existing private key
+description: How to start an IPFS node with a pre-generated private key.
+---
+
+# Use an existing private key
+
+There are two locations in which IPFS stores private keys:
+
+1. In the config file (`~/.ipfs/config`), following the [config specification](https://github.com/ipfs/go-ipfs/blob/master/docs/config.md#identityprivkey) to store the default private key.
+2. In the Keystore (`~/.ipfs/keystore`), following the [fs-repo specification](https://github.com/ipfs/specs/blob/master/REPO_FS.md) to store all other private keys.
+
+Private keys are handled programmatically through code or via the CLI. Here we will show you how to use your existing private keys using both methods.
+
+## CLI (`go-ipfs`)
+
+There's no way of directly initializing an IPFS node using your private keys from IPFS's Go CLI at this moment. To learn more and look for possible workarounds, see the discussion on issue [ipfs/go-ipfs#4240](https://github.com/ipfs/go-ipfs/issues/4240).
+
+You can, however, import private keys into the Keystore:
```

Comment:
```suggestion
However, you can import private keys into the keystore:
```

Reply:
Keystore here refers to a [specific feature of IPFS](https://github.com/ipfs/specs/blob/master/KEYSTORE.md). Shouldn't we keep that capitalized and treat it like a proper noun to keep users aware that we are referring to that particular IPFS feature and not some other generic keystore?

Reply:
Ah, sorry I missed this one. I assumed we were talking about the generic _keystore_. In this case, we should use something like _IPFS keystore_:

```suggestion
You can, however, import private keys into the IPFS keystore:
```